src/raster/r.area.createweight/r.area.createweight.py

```diff
@@ -0,0 +1,1464 @@
+#!/usr/bin/env python3
+#
+"""
+# MODULE:       r.area.createweight
+#
+# AUTHOR(S):    Grippa Tais, Safa Fennia, Charlotte Flasse
+#
+# PURPOSE:      Create a weighting layer for dasymetric mapping,
+#               using a random forest regression model.
+#
+# DATE:         2021/08/05
+#
+# COPYRIGHT:    (C) 2021 Grippa Tais, Safa Fennia, Charlotte Flasse
+#               and by the GRASS Development Team
+#
+#               This program is free software under the
+#               GNU General Public License (>=v2).
+#               Read the file COPYING that comes with GRASS
+#               for details.
+"""
+
+#%module
+#% description: Create a dasymetric weighting layer with Random Forest
+#% keyword:Raster
+#% keyword:Statistics
+#% keyword:Density
+#% keyword:Dasymetry
+#% keyword:Resample
+#%end
+#%option G_OPT_V_MAP
+#% key: vector
+#% label: Vector with spatial units
+#% description: Polygon vector containing unique ID and response
+#               variable in the attribute table
```

Comment:
Ha, this works? Cool if it does. Please test and let me know.

Reply:
What do you mean ? The G_OPT_V_MAP option ? At least on GRASS 7.8.5 it works

Reply:
Multi-line description.

I was surprised to see this:

```
#% description: Polygon vector containing unique ID and response
#               variable in the attribute table
```

as opposed to this:

```
#% description: Polygon vector containing unique ID and response variable in the attribute table
```

Perhaps a hidden feature?

Reply:
Indeed you are right :) and the multi-line indeed do not work on display. I will fix it directly.